apps/files/src/components/Files-App.vue

```diff
@@ -1,209 +1,225 @@
-							li
-								label
-									input.uk-checkbox(type='checkbox', name='Pending', v-model="filterBy.hidden")
-									span(:class="{ 'uk-text-primary' : filterBy.hidden }").uk-text-meta.uk-margin-small-left Hidden files
-			div(uk-grid).uk-grid-collapse
-				main.uk-width-expand._scroll_container
-					table.file-table-header.uk-table.uk-table-middle.uk-table-divider.uk-margin-remove
-						thead
-							tr
-								th
-									input(type="checkbox").uk-checkbox.uk-margin-small-left
-								th Name
-								th(class="uk-visible@l").uk-text-right Owner
-								th Size
-								th(class="uk-visible@s") Date
-						tbody
-							tr(v-for="(file, id) in files", v-if="ifFiltered(file)", :data-file-id="file._id", :class="{ '_is-selected' : isChecked(file) }").uk-animation-fade
-								td.uk-table-shrink
-									input(type="checkbox", :checked="isChecked(file)", @click="toggleFileSelect(file)").uk-checkbox.uk-margin-small-left
-
-								// --- Name ----------
-								td(v-if="!file.extension", @click="toggleFileSelect(file)").uk-text-truncate.uk-visible-toggle
-									drag(:transfer-data="file")
-										drop(@drop="onDrop('file-list', file, ...arguments)")
-											a(@click.stop="routerLink(file.path)").uk-link-text.uk-position-relative
-												i.material-icons.uk-text-primary.uk-position-center-left {{ file.type }}
-												span {{ file.name }}
-
-								td(v-else).uk-text-truncate(@click="toggleFileSelect(file)")
-									a(@click.stop="endOfDummy").uk-link.uk-position-relative
-										i.material-icons.uk-text-primary.uk-position-center-left {{ file.type }}
-										span {{ file.name }}
-									span.uk-text-meta .{{ file.extension }}
-
-								// --- Owner ----------
-								td(class="uk-visible@l").uk-text-nowrap.uk-table-shrink.uk-text-right
-									div(v-if="file.sharedIn").uk-text-meta.uk-inline
-										span(uk-tooltip, :title="file.sharedIn.mail") {{ file.sharedIn.name }}
-
-								// --- Size ------------------
-								td.uk-text-nowrap.uk-table-shrink
-									span.uk-text-meta {{ file.size | fileSize }}
-
-								// --- Filedate ------------------
-								td(class="uk-visible@s").uk-text-nowrap.uk-table-shrink
-									time.uk-text-meta {{ file.mdate | formDateFromNow }}
-
-				aside.uk-width-medium.uk-background-default.uk-padding-small(v-show="selected.length > 0", class="uk-width-large@l uk-padding@l").uk-animation-slide-right-small
-					FileDetails(@reset="resetFileSelection")
+<template>
+	<v-container id="files-app" fluid class="pa-0">
+		<v-toolbar class="elevation-1">
+ 			<v-btn v-if="!createFolder" @click="createFolder = !createFolder" flat><v-icon large>create_new_folder</v-icon></v-btn>
+			<v-btn v-if="createFolder" @click="addNewFolder(newFolderName)" flat><v-icon large>add</v-icon></v-btn>
+			<v-flex v-if="createFolder" xs2>
+				<v-text-field
+					placeholder="New folder Name"
```

Comment:
translations missing - right?

Reply:
yes - will bring them in them in one of the next commits